Tribler/Test/tools.py

```diff
@@ -0,0 +1,54 @@
+# Shameless copypaste from Nosetests package
```

Comment:
There is sort of a copyright on this code then which we should respect. Why not just use the functionality offered by nose in the first place?

Reply:
The reason for this copypaste is to become independent from Nose package. However, as we still use Nose as our top-level test runner, we can remove it and re-add the Nose one.
Regarding the copyright, there is always an option to just copy their copyright message into the source file.

What would you prefer?

Reply:
Unless we run into nasty versioning issues, I'd prefer re-adding the nose library calls.

Reply:
rolled back nose-stuff